packages/xs-extra-dummy/upstream-extra-dummy.master/opam

```diff
@@ -5,13 +5,13 @@ authors:        "Christian Lindig <christian.lindig@citrix.com>"
 homepage:       "https://github.com/xapi-project/xs-opam"
 license:        "BSD"
 depends: [
+  "merlin"
   "ocaml-system"
-  "ocp-indent"
+  "ocamlformat"
+  "odig"
   "travis-opam"
-  "merlin"
-  "xenctrl" { = "dummy"}
-  "ocveralls"
   "utop"
+  "xenctrl" { = "dummy"}
 
   # TODO: not used but we may want to keep
   "opasswd"
```

Comment:
could we remove this too?

Reply:
I think we could remove it as well. But given that @psafont is not here, I'd prefer to address is another time.

Reply:
@edwintorok added it a couple of years ago, don't know for what purpose

Reply:
I think the dummy package might be needed in our internal build system, where we want to use the system provided xenctrl (built and installed outside of opam), instead of the version that we would build here.
The opam version of xenctrl is only used in the CI, and for local builds, not for the production builds.

Reply:
I was talking about opasswd rather than xenctrl

Reply:
Ah, that was there when we were testing @lindig's unixpwd implementation vs upstream `opasswd`. `unixpwd` is now inside xapi, so this can be dropped, I'll send a PR.